### PR TITLE
VCSWP-15719: Added active property to CallsSearchFilters.java

### PR DESCRIPTION
**Pull request by jbohnevail**
In the class CallsSearchFilters, added the variable `private boolean active;` as well as getters and setters for it.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 None
+<a name="4.1.3"</a>
+## [3.0.2] 2021-11-3
```
Comment (ngarg-vail) [inline CHANGELOG.md:10]: This is not the correct value for the version - should be `4.1.3`
In addition we need to update the actual version number in the `build.gradle` file

**Review by ngarg-vail — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (jbohnevail) [reply CHANGELOG.md:10]: You're referring to the version in square brackets correct?

Comment (ngarg-vail) [reply CHANGELOG.md]: Correct

Comment (jbohnevail) [reply CHANGELOG.md]: Ok cool went ahead and updated that version along with the version in the `build.grandle` file

**Review by ngarg-vail — APPROVED** [review]

**Pull request merged**